### PR TITLE
Fix output path

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
I'm running into an issue where I would like to add a version number to my output files (in case a file gets corrupted/changes, etc) however I'm currently using `dataset_path` as the prefix of my output file, since `output_path` changes per cell type and I only need to generate this file once. I've added in a version to the output table, which is taken from the input-prefix.

@lgruen this is a crude way of fixing this. Please let me know if you think there's a better way

### CODE REVIEW

File: scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py [hunk 1]
```diff
@@ -136,8 +136,10 @@ def submit_eqtl_jobs(
                 )
 
                 keys = os.path.join(input_path, 'OneK1K_CPG_IDs.tsv')
+                version = output_dir.split('/')[-1]
```
Comment (lgruen) [inline scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py]: An alternative approach would be to not launch this script with the output path containing the cell type, but instead rely on the script to append the cell type to the path itself when you write outputs. Does that make sense?

File: scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py [hunk 2]
```diff
@@ -136,8 +136,10 @@ def submit_eqtl_jobs(
...
+                # put version number at end of output path
                 output_prefix = os.path.join(
-                    output_dir, f'{cell_type}', f'chr{chromosome}'
+                    output_dir.strip(version), f'{cell_type}', f'chr{chromosome}', f'{version}'
```
Comment (lgruen) [inline scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py]: `strip` is probably not what you want here, but `removesuffix`. (See https://stackabuse.com/python-remove-the-prefix-and-suffix-from-a-string/.)

**Review by lgruen — CHANGES_REQUESTED** [review]

File: scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py (hunk 1 above)
Comment (KatalinaBobowik) [reply scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py]: Yes, that's a good idea. And would look a lot cleaner so I could just use `output_path` instead

File: scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py (hunk 2 above)
Comment (KatalinaBobowik) [reply scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py]: neat, I didn't know about `removesuffix`

**Review by lgruen — APPROVED** [review]

**Pull request merged**